aws/provider_test.go

```diff
@@ -1190,6 +1183,44 @@ func testAccPreCheckSkipError(err error) bool {
 	return false
 }
 
+type testSweepResource struct {
```

Comment:
Would it make sense to move the sweeper functions to https://github.com/hashicorp/terraform-provider-aws/blob/main/aws/aws_sweeper_test.go?

Reply:
Moved, along with some other sweep stuffs.